example/spec/parallel/dbRecords/DeadLetterArchiveProcessingSpec.js

```diff
@@ -0,0 +1,178 @@
+const { postDeadLetters } = require('@cumulus/api-client/deadLetters');
+const { waitForGranule } = require('@cumulus/api-client/granules');
+const { createProvider, deleteProvider } = require('@cumulus/api-client/providers');
+const { deleteRule } = require('@cumulus/api-client/rules');
+const {
+  loadCollection,
+} = require('@cumulus/integration-tests');
+const { findExecutionArn } = require('@cumulus/integration-tests/Executions');
+const { createOneTimeRule } = require('@cumulus/integration-tests/Rules');
+const { getStateMachineArnFromExecutionArn } = require('@cumulus/message/Executions');
+const { constructCollectionId } = require('@cumulus/message/Collections');
+const { randomString } = require('@cumulus/common/test-utils');
+const { putJsonS3Object } = require('@cumulus/aws-client/S3');
+const { waitForListObjectsV2ResultCount } = require('@cumulus/integration-tests');
+
+const { loadConfig } = require('../../helpers/testUtils');
+
+describe('A dead letter record archive processing operation', () => {
+  let beforeAllFailed = false;
+  let stackName;
+  let systemBucket;
+  let testCollection;
+  let testProvider;
+  let testRule;
+  let testGranule;
+  let archivePath;
+  let messageKey;
+
+  beforeAll(async () => {
+    try {
+      const ingestTime = Date.now() - 1000 * 30;
+
+      const config = await loadConfig();
+      stackName = config.stackName;
+      systemBucket = config.bucket;
+
+      let response;
+      const testId = randomString();
+
+      testCollection = await loadCollection({
+        filename: './data/collections/s3_MOD09GQ_006/s3_MOD09GQ_006.json',
+        postfix: testId,
+      });
+      response = await createCollection({ prefix: stackName, collection: testCollection });
+      if (response.statusCode !== 200) {
+        throw new Error(`createCollection failed: ${JSON.stringify(response)}`);
+      }
+
+      testProvider = {
+        id: `s3_provider_${testId}`,
+        host: 'cumulus-sandbox-fake-s3-provider',
+        protocol: 's3',
+        globalConnectionLimit: 1000,
+      };
+      response = await createProvider({ prefix: stackName, provider: testProvider });
+      if (response.statusCode !== 200) {
+        throw new Error(`createProvider failed: ${JSON.stringify(response)}`);
+      }
+
+      const testFiles = [
+        fakeFileFactory({ bucket: systemBucket }),
+        fakeFileFactory({ bucket: systemBucket }),
+      ];
+
+      testGranule = fakeGranuleFactoryV2({
+        granuleId: `MOD09GQ.${randomString()}.hdf`,
+        collectionId: constructCollectionId(testCollection.name, testCollection.version),
+        files: testFiles,
+      });
+
+      testRule = await createOneTimeRule(
+        stackName,
+        {
+          workflow: 'HelloWorldWorkflow',
+          collection: pick(testCollection, ['name', 'version']),
+          provider: testProvider.id,
+          payload: { testId },
+        }
+      );
+      console.log('createOneTimeRule:', testRule);
```

Comment:
Was this intentionally left in?

Reply:
No!

Reply:
https://github.com/nasa/cumulus/pull/2181/commits/2682e4efa888a2f54fd8fbd254d924c6e7bd3c7c